Embedded/Arduino/Motor_controller/src/Hardware/Battery/BatteryHealth.cpp

```diff
@@ -14,95 +18,83 @@ int buzzerPulseCount = 0;
 unsigned long lastWarningTriggerTime = 0;
 
 /*Variabler för LastSignal Start funktionen*/
-
 // ska vara false från början
 bool lastSignalActive = false;
 unsigned long lastSignalStartTime = 0;
 
+// Variabler för att skicka batteri status till servern
+bool warningSignal = false;
+bool sleepModeSigna = false; 
+
 // Läser av det analoga värdet från A0 och räknar ut laddningen av batteriet
-float CalculateBatteryHealth(int analogPin, float Vref)
-{
+float CalculateBatteryHealth(int analogPin, float Vref) {
```

Comment:
According to your coding guidelines bracket should be placed one line below, is this correct? If so there are many instances we need to change here
```suggestion
float CalculateBatteryHealth(int analogPin, float Vref) 
{
```